packages/components/menu/src/menu-button.ts

```diff
@@ -51,6 +51,9 @@ export class MenuButton extends ScopedElementsMixin(LitElement) {
   /** The fill of the button. */
   @property() fill: ButtonFill = 'outline';
 
+  /** If the menu button is icon only, use this to set the `aria-label` on the button. */
+  @property() label?: string;
```

Comment:
Are we sure that this is really necessary? Adding `aria-label` on the `menu-button` is enough for the Voice Over, like in this example:
https://proud-water-0d02ee203-1541.westeurope.2.azurestaticapps.net/categories/components/menu/usage/

Maybe this PR should only cover adding `aria-label` in the stories  of menu button - icon only?

Reply:
If we really want to keep it this way, we also need to change the documentation about `aria-label` https://proud-water-0d02ee203-1541.westeurope.2.azurestaticapps.net/categories/components/menu/accessibility/ in the menu button.

Reply:
Agreed; if we force the use of `aria-label` we sort of remove the option of using `aria-labeled-by` when that is more appropriate.
If a dev doesn't think about using `aria-label` they probably also won't think to use `label`

Reply:
Changed the PR to now only add an `aria-label` to the icon only stories